src/main/java/net/minecraftforge/registries/ForgeRegistries.java

```diff
@@ -127,7 +127,8 @@
         // Vanilla Dynamic
         public static final ResourceKey<Registry<Biome>> BIOMES = key("worldgen/biome");
 
-        //Forge
+        // Forge
+        // New forge-added registries should be populated in VanillaConnectionNetworkFilter#FORGE_ADDED_REGISTRIES
         public static final ResourceKey<Registry<DataSerializerEntry>> DATA_SERIALIZERS = key("data_serializers");
```

Comment:
Is there a reason this isn't in forge's domain? If not it may make sense to add a TODO for 1.19 to move it to forge's domain and remove having to add them to `FORGE_ADDED_REGISTRIES` and instead just check if the namespace is forge's? (Or would that set still make sense to have for purposes if forge ever decides to add any registries itself that supports tags)